com.ticktick.TickTick.yml

```diff
@@ -0,0 +1,58 @@
+app-id: com.ticktick.TickTick
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: ticktick
+rename-desktop-file: ticktick.desktop
+rename-icon: ticktick
+
+tags:
+  - proprietary
+
+finish-args:
+  - --socket=x11
+  # needs electron v11 (chromium v87) or newer with ozone enabled
+  # https://github.com/electron/electron/issues/10915
+  #- --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: ticktick
+    buildsystem: simple
+    build-commands:
+      - install -dm755 ${FLATPAK_DEST}/TickTick
+      - |
+        bsdtar -Oxf ticktick.deb 'data.tar*' |
+        bsdtar -xf - \
+        -C ${FLATPAK_DEST}/TickTick \
+        --strip-components=3 \
+        ./opt/TickTick
+      - |
+        bsdtar -Oxf ticktick.deb 'data.tar*' |
+        bsdtar -xf - \
+        -C ${FLATPAK_DEST}/ \
+        --strip-components=2 \
+        ./usr
+
+      - desktop-file-edit --set-key=Exec --set-value=ticktick ${FLATPAK_DEST}/share/applications/ticktick.desktop
+      - install -Dm755 ticktick -t ${FLATPAK_DEST}/bin/
+
+    sources:
+      - type: file
```

Comment:
This should be `extra-data` as this is a proprietary application we have no rights to redistribute.